### ISSUE TITLE
Android pinch to zoom not working 

### ISSUE THREAD

**Issue by mink-ang** [jobtoday/react-native-image-viewing#62]
If you have swipe to dismiss enabled with pinch to zoom, they interfere with each other and creates a weird experience. Can't really zoom and very janky. I am using a pixel 2. Then you have that issue with image zoomed to max after zooming once and only gets cleared after swiping to other images a few times.

**Comment by suretarget** [comment]
I can confirm also the bug. If I disable swipe to dismiss it's working fine. Anyone find a workaround yet?

**Comment by Stukz** [comment]
@suretarget same here v0.2.0

**Comment by jhclaura** [comment]
I can't make it zoom in Android even after setting `swipeToCloseEnabled` to `false`. Any guidance appreciated!

**Comment by karanmartian** [comment]
exactly, its a really pissing bug on android. +1. Works well on ios but not on android, it feels like its stuck and sluggish on android. u have to try really hard to zoom and it zooms very little on android. on ios its perfect, so fluid.
